uobtheatre/bookings/templates/emails/booking_confirmation_email.html

```diff
@@ -0,0 +1,6 @@
+<p>Hello {{ user_name }}!</p>
+
+<p> Your booking for {{ production_name }} on {{ start|date:"l, d F Y" }} at {{ start|time:"H:i" }} is confirmed!</p>
```

Comment:
Could we make this a bit nicer? Some graphics or something?

Reply:
I think that's out of the scope of this ticket. This is purely function we should absolutely make a better email with some nice images, better copy and maybe even with the ticket emebedded. I suggest we wait for designs rather than wasting taking making a nice email we know we should throw away